code/modules/uplink/uplink_items.dm

```diff
@@ -990,8 +986,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 0
 	purchasable_from = UPLINK_CLOWN_OPS
 
-//SKYRAT EDIT REMOVAL BEGIN
-/*
 /datum/uplink_item/explosives/detomatix
```

Comment:
Lol no this isn't getting back in. Even as TC trades, they're just a headache that waaaay too easily allows someone to fuck someone else's round with 0 roleplay whatsoever.